src/pages/audio-rewards-page/store/sagas.ts

```diff
@@ -241,6 +241,12 @@ function* claimChallengeRewardAsync(
             break
           case FailureReason.BLOCKED:
             throw new Error('User is blocked from claiming')
+          // For these 'attestation aggregation errors',
+          // we've already retried in libs so unlikely to succeed here.
+          case FailureReason.MISSING_CHALLENGES:
+          case FailureReason.CHALLENGE_INCOMPLETE:
+            yield put(claimChallengeRewardFailed())
+            break
           case FailureReason.UNKNOWN_ERROR:
```

Comment:
should we add a default back?

Reply:
good idea. I think UNKNOWN and default should probably handle the same